samples/excel/06-named-item/create-and-remove-named-item.yaml

```diff
@@ -0,0 +1,182 @@
+id: excel-06-named-item-create-and-remove-named-item-
+name: Create a named item
+description: Create a named item for a formula
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#add-name").click(addName);
+        $("#remove-name").click(removeName);
+        $("#setup").click(setup);
+
+        async function addName() {
+            try {
+                await Excel.run(async (context) => {
+
+                    const sheet = context.workbook.worksheets.getItem("Sample");
+
+                    sheet.names.add("TotalAmount", "=SUM(ExpensesTable[AMOUNT])");
+
+                    sheet.getRange("D11").values = [["=TotalAmount"]];
+
+                    sheet.getUsedRange().format.autofitColumns();
```

Comment:
The table doesn't look good after removal of these lines. Let's leave it in....it's included in the setup function, which is separate from the named item code we want to highlight. 

FYI - It looks like you're looking at an older check-in. Can you look at the latest check-in please? Thanks!